intertidal/elevation.py

```diff
@@ -1264,22 +1264,24 @@ def intertidal_cli(
         if exposure_offsets:
             log.info(f"{run_id}: Calculating Intertidal Exposure")
 
-            # Select times used for exposure modelling
-            all_times = pd.date_range(
-                start=round_date_strings(start_date, round_type="start"),
-                end=round_date_strings(end_date, round_type="end"),
-                freq=modelled_freq,
-            )
-
             # Calculate exposure
-            ds["exposure"], tide_cq = exposure(
+            exposure_filters, tide_cq_dict = exposure(
```

Comment:
In the function below, `tide_cq_dict` is called `modelledtides_ds` - can we use that terminology here too for consistency? 

(are these raw modelled tides or tide quantiles? Alternatively, we could call it something like `tide_quantiles_ds` too if that was more clear)

Reply:
I'll update that variable name. It's a legacy hangover from an earlier version of the code.